freebsd-11.1-amd64.json

```diff
@@ -0,0 +1,119 @@
+    "http_directory": "http",
+    "boot_wait": "10s",
+    "boot_command": [
+      "2<enter><wait10><wait10><wait10>",
+      "<enter><wait>",
+      "mdmfs -s 100m md1 /tmp<enter><wait>",
+      "dhclient -l /tmp/dhclient.leases -p /tmp/dhclient.pid em0<enter><wait5>",
+      "fetch -o /tmp/installerconfig http://{{.HTTPIP}}:{{.HTTPPort}}/freebsd-11.1/installerconfig<enter><wait>",
+      "bsdinstall script /tmp/installerconfig && reboot<enter>"
+    ],
+    "ssh_timeout": "{{user `ssh_timeout`}}",
+    "ssh_username": "vagrant",
+    "ssh_password": "vagrant",
+    "guest_additions_mode": "disable",
+    "shutdown_command": "sudo poweroff",
+    "vboxmanage": [
+      ["modifyvm", "{{.Name}}", "--memory", "{{user `memory`}}"],
+      ["modifyvm", "{{.Name}}", "--cpus", "{{user `cpus`}}"]
+    ],
+    "post_shutdown_delay": "30s"
+  }, {
+    "type": "vmware-iso",
+    "guest_os_type": "freeBSD-64",
+    "iso_url": "{{user `mirror`}}/releases/ISO-IMAGES/11.1/FreeBSD-11.1-RELEASE-amd64-disc1.iso",
+    "iso_checksum": "{{user `iso_checksum`}}",
+    "iso_checksum_type": "{{user `iso_checksum_type`}}",
+    "output_directory": "output-freebsd-11.1-amd64-{{build_type}}",
+    "vm_name": "packer-freebsd-11.1-amd64",
+    "disk_size": "{{user `disk_size`}}",
+    "headless": "{{user `headless`}}",
+    "http_directory": "http",
+    "boot_wait": "10s",
+    "boot_command": [
+      "2<enter><wait10><wait10><wait10>",
+      "<enter><wait>",
+      "mdmfs -s 100m md1 /tmp<enter><wait>",
+      "dhclient -l /tmp/dhclient.leases -p /tmp/dhclient.pid em0<enter><wait5>",
+      "fetch -o /tmp/installerconfig http://{{.HTTPIP}}:{{.HTTPPort}}/freebsd-11.1/installerconfig<enter><wait>",
+      "bsdinstall script /tmp/installerconfig && reboot<enter>"
+    ],
+    "ssh_timeout": "{{user `ssh_timeout`}}",
+    "ssh_username": "vagrant",
+    "ssh_password": "vagrant",
+    "tools_upload_flavor": "freebsd",
+    "shutdown_command": "sudo poweroff",
+    "vmx_data": {
+      "memsize": "{{user `memory`}}",
+      "numvcpus": "{{user `cpus`}}"
+    }
+  }],
+  "provisioners": [{
+    "type": "shell",
+    "scripts": [
+      "scripts/freebsd/virtualbox.sh",
+      "scripts/freebsd/vmware.sh",
+      "scripts/freebsd/init.sh",
+      "scripts/common/vagrant.sh",
+      "scripts/common/sshd.sh",
+      "scripts/freebsd/cleanup.sh",
+      "scripts/freebsd/minimize.sh"
+    ]
+  }],
+  "post-processors": [{
+    "type": "vagrant",
+    "compression_level": "{{user `compression_level`}}",
+    "output": "freebsd-11.1-amd64-{{.Provider}}.box",
+    "vagrantfile_template": "vagrantfile_templates/freebsd.rb"
+  }],
+  "variables": {
+    "compression_level": "6",
+    "cpus": "1",
+    "disk_size": "40000",
+    "headless": "false",
+    "iso_checksum": "ff4c749ea0aaaceedb2432ba3e0fd0c1b64f5a72141b1ec06b9ced52b5de0dbf",
+    "iso_checksum_type": "sha256",
+    "memory": "512",
+    "mirror": "http://ftp.freebsd.org/pub/FreeBSD",
```

Comment:
the officially announced URL `https://download.freebsd.org/ftp/releases/ISO-IMAGES/11.1/` should be used.